src/odemis/acq/stream/_static.py

```diff
@@ -124,6 +155,48 @@ def _init_thread(self):
         '''
         pass
 
+    def _updateImage(self):
```

Comment:
Is this ever used? IIRC, the Static2DStream is always projected via a Projection, which doesn't use `_updateImage()`. So, it's might be fine to completely remove this method. Otherwise, I'd suggest to just modify the parent method to handle zIndex itself, instead of duplicating the whole code, for ~2 lines change.